test/e2e/common_test.go

```diff
@@ -193,8 +192,6 @@ var _ = SynchronizedBeforeSuite(func() []byte {
 })
 
 func (p *PodmanTestIntegration) Setup() {
-	cwd, _ := os.Getwd()
-	INTEGRATION_ROOT = filepath.Join(cwd, "../../")
```

Comment:
Any reason not to nuke this now-empty function?

Reply:
no good to remove, and thanks to https://github.com/containers/podman/commit/ab29ff2f66216cfafa01add2c3238831a3824317 it only needs to happen in one place.